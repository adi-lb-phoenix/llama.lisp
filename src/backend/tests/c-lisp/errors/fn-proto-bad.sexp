(c-lisp
    (define (main int)
        (ret 0)))
