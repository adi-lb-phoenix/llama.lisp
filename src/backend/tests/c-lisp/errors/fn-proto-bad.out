CodegenError: Bad function prototype: ['main', 'int']
